xgboost_ames_housing/housing.py

```diff
@@ -21,7 +21,7 @@
 from sklearn.preprocessing import Imputer
 from xgboost import XGBRegressor
 
-def read_input(file_name, test_size=0.25):
+def read_input(file_name, test_size=0.25, shuffle=True):
```

Comment:
Can we make shuffle=False as the default value?

It makes the model output deterministic and some integration test may rely on it.